manifests/p/Postbox/Postbox/7.0.51/Postbox.Postbox.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Postbox.Postbox
+PackageVersion: 7.0.51
+PackageLocale: en-US
+Publisher: Postbox, Inc.
+PublisherUrl: https://www.postbox-inc.com
+PublisherSupportUrl: https://support.postbox-inc.com/hc/en-us

```

```suggestion
#PublisherSupportUrl: https://support.postbox-inc.com/hc/en-us
```
For now, let's comment this out for the PR to pass.